js/src/hooks/usePolicyCheck.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import useAppSelectDispatch from './useAppSelectDispatch';
+
+/**
+ * Get allowed countries from calling `useAppSelectDispatch` with `"getAllowedCountries"`.
+ * Returns `{ hasFinishedResolution, data, invalidateResolution }`.
+ *
+ * `data` is an object of country mapping. e.g.:
+ *
+ * ```json
+ * {
+ * 		 "AR": {
+ * 			"name": "Argentina",
+ * 			"currency": "ARS"
+ * 		}
+ * }
+ * ```
+ */
+const usePolicyCheck = () => {
+	return useAppSelectDispatch( 'usePolicyCheck' );
```

Comment:
useAppSelectDispatch is expecting as first parameter the selector's name, in this case should be `getPolicyCheck`